File: backend/src/main/java/backend/repository/entity/PfConfig.java
package backend.repository.entity;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Data;

/** A JPA entity corresponding to pf_config table */
@Entity
@Table(name="pf_config")
@Data
public class PfConfig {
	@Id private final Long id = 1L;
	private String host = "127.0.0.1";
	private Integer port = 22;
	private String username = "username";
	private String password = "password";
}

File: backend/src/main/kotlin/backend/repository/entity/PfConfig.kt
package backend.repository.entity

import lombok.Data
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

/** A JPA entity corresponding to pf_config table  */
@Entity
@Table(name = "pf_config")
data class PfConfig (
    @Id var id: Long = 1L,
    var host: String = "127.0.0.1",
    var port: Int = 22,
    var username: String  = "username",
    var password: String = "password"
)
